src/flush-microtasks.js

```diff
@@ -59,7 +62,11 @@ export default function flushMicroTasks() {
         jest.advanceTimersByTime(0)
         resolve()
       } else {
-        enqueueTask(resolve)
+        scheduleCallback(null, () => {
+          enqueueTask(() => {
+            resolve()
+          })
+        })
```

Comment:
With the change above, we no longer need this first argument

```suggestion
        scheduleCallback(() => enqueueTask(resolve))
```